packages/cactus-plugin-satp-hermes/src/test/typescript/unit/config-validating-functions/validate-cc-config.test.ts

```diff
@@ -0,0 +1,656 @@
+          rpcApiWsHost: "ws://127.0.0.1:7545",
+        },
+        claimFormats: [2],
+      },
+    ];
+    expect(() =>
+      validateCCConfig({
+        configValue: {
+          oracleConfig,
+          bridgeConfig,
+        },
+      }),
+    ).toThrowError(/Invalid config\.bridgesConfig && config\.oracleConfig/);
+  });
+
+  it("should throw when is an invalid FabricConfig", async () => {
+    const badFabricConfig = {
+      networkIdentification: {
+        id: "FabricLedgerTestNetwork",
+        ledgerType: "FABRIC_2",
+      },
+      userIdentity: {
+        credentials: {
+          certificate:
+            "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----\n",
+          privateKey:
+            "-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n",
+        },
+        mspId: "Org2MSP", // Membership Service Provider ID
+        type: "X.509", // Credential type; typically X.509 for Fabric
+      },
+      connectorOptions: {
+        dockerBinary: "/usr/local/bin/docker",
+        peerBinary: "/fabric-samples/bin/peer",
+        goBinary: "/usr/local/go/bin/go",
+        cliContainerEnv: FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_1,
+        sshConfig: {
+          // SSH access details (if interacting over SSH)
+          host: "172.20.0.6",
+          privateKey:
+            "-----BEGIN OPENSSH PRIVATE KEY-----\n...\n-----END OPENSSH PRIVATE KEY-----\n",
+          username: "root",
+          port: 22,
+        },
+        connectionProfile: connectionProfile,
+        discoveryOptions: {
+          enabled: true,
+          asLocalhost: true,
+        },
+        eventHandlerOptions: {
+          strategy: "NETWORK_SCOPE_ALLFORTX",
+          commitTimeout: 300,
+        },
+      },
+      channelName: "mychannel",
+      targetOrganizations: [
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_1,
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_2,
+      ],
+      caFile:
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_2.ORDERER_TLS_ROOTCERT_FILE,
+      ccSequence: 1,
+      orderer: "orderer.example.com:7050",
+      ordererTLSHostnameOverride: "orderer.example.com",
+      connTimeout: "60",
+      signaturePolicy: "config.signaturePolicy",
+      mspId: "Org2MSP",
+      wrapperContractName: "exampleWrapperContractName",
+      leafId: "exampleLeafId",
+      keyPair: {
+        publicKey:
+          "0x03a34e1d66b78e47fa1bba3445a6019acb5b9c87d0c6ad81c09e7d496682ae81fc",
+        privateKey:
+          "0x1c3a1e987fdabc9aeea47e457ad4db9732f7d7b6d2856d6e3f9b8d88ff6e227e",
+      },
+      claimFormats: [2],
+    };
+    const bridgeConfig = badFabricConfig;
+    expect(() =>
+      validateCCConfig({
+        configValue: {
+          bridgeConfig,
+        },
+      }),
+    ).toThrowError(/Invalid config\.bridgesConfig \|\| config\.oracleConfig/);
+  });
+
+  it("should throw when is an invalid FabricOptionsJSON", async () => {
+    const badFabricConfig = {
```

Comment:
In these cases, just for developer experience, add a small comment saying what is wrong with this Fabric config, because it is not really understandable what's missing